bin/pycbc_make_html_page

```diff
@@ -214,6 +219,12 @@ for cwd in dirs:
     cwd.files.sort(key=lambda x: x.path)
 dirs_0.sort(key=lambda x: x.title())
 
+local_logo = opts.logo[:4] != "http"
+if local_logo:
+    logo_ext = opts.logo.split(".")[-1]
+    logo_loc = opts.output_path + '/logo.' + logo_ext
+    shutil.copy2(opts.logo, logo_loc)
```

Comment:
In the HTML template the substitution is only ``{logo}`` which seems appropriate. But why is there so much work being done to rename the image to ``logo + logo_ext``? Why not use ``os.path.basename`` and then copy ``shutil.copy2`` and keep the original file? It seems a lot of this code could be removed.

Reply:
I thought that it may be better to have logo being standardly named so that it would be more easily identified within the directory, but that doesn't make sense if allowing different extensions

using os.path.basename now as suggested